WebContent/addRestaurant/addRestaurant.js

```diff
@@ -0,0 +1,9 @@
+var cuisines ="";
```

Comment:
So we will need to do something similar for the notes section.  If you can think of any other pieces of information that need to be formatted before submission, you can process them here.